src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="form-search form-big"
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <label class="form-label form-label-big">
```

Comment:
label redundant

Reply:
should I replace all 'labels' by <div>?

Reply:
I tried to use 'outline:none' for input, also for 'form','form:focus','form:active', but there are no changes -the outline still exist and default value of my brauser is dark-grey. what do you recommend to do?

Reply:
Yep, if you have no label (no text) you may use `div`.